lib/ws.js

```diff
@@ -47,11 +147,7 @@ class BinanceWS {
             eventHandler(event);
         });
 
-        ws.on('error', () => {
-            // node.js EventEmitters will throw and then exit if no error listener is registered
-        });
-
-        return ws;
+        return socket.connect();
```

Comment:
This no longer returns a WebSocket - it may be risky changing the return value of such a high level method, so would prefer it remained consistent.